pkg/controller/infrastructureconfig/sync_specstatus.go

```diff
@@ -0,0 +1,195 @@
+package infrastructureconfig
+
+import (
+	"fmt"
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/cluster-network-operator/pkg/util/ip"
+	"log"
+)
+
+type specStatusSynchronizer interface {
+	SpecStatusSynchronize(*configv1.Infrastructure) (*configv1.Infrastructure, error)
+}
+
+type specStatusSunchronizer struct{}
```

Comment:
typo? `Sunchronizer`?  Or a naming collision workaround?  If it's intentional then it might need a comment, or it will trigger future pendants.

Reply:
Yes, typo indeed, will fix